saleor/plugins/manager.py

```diff
@@ -448,7 +448,7 @@ def calculate_checkout_line_unit_price(
             default_value * quantity,
         )
         default_taxed_value = TaxedMoney(
-            net=total_value / quantity, gross=default_value
+            net=total_value / quantity, gross=total_value / quantity
```

Comment:
Isn't something that can cause incorrect values on our side? Shouldn't we patch them also in previous versions?

Reply:
It is a bug that was found during development of this PR... maybe the best way would be to fix this particular line in separate PR and port it to versions affected

Reply:
To me it looks risky, I would remove and fix in a separate PR, with proper testing and QA only of this single case.

Reply:
Also with an GH issue first that proves and shows why and when this calculation is incorrect.